lupa-core/src/main/kotlin/org/jetbrains/research/lupa/kotlinAnalysis/util/RepositoryOpenerUtil.kt

```diff
@@ -15,69 +15,55 @@ class RepositoryOpenerUtil {
         private val logger: Logger = Logger.getInstance(RepositoryOpenerUtil::class.java)
 
         /**
-         * Opens and reloads each project in given dataset by [path][datasetPath] and then
+         * Opens and reloads project in given directory by [path][repositoryRoot] and then
          * runs [action] on opened project.
-         * Also runs [repositoryPostAction] on each repository after processing it
          * This opening method allow to get project structure (all modules), but is more time consuming then
          * [standardRepositoryOpener].
          */
         fun openReloadRepositoryOpener(
```

Comment:
Should you use Java\Kotlin prefix in the name if this opener works only for them?